.flowconfig

```diff
@@ -5,7 +5,7 @@
 [include]
 
 [libs]
-interface/
+src/flow/interfaces/
```

Comment:
I would keep interfaces in the root of the repo... it's really uncommon to do it that way